conductr_cli/sandbox_features.py

```diff
@@ -27,6 +28,73 @@ def __init__(self, name, port):
         self.port = port
 
 
+class ProxyingFeature:
+    """Proxying feature.
+
+    On start, the proxying features will be run.
+    """
+
+    name = 'proxying'
+    ports = []
+    dependencies = []
+
+    def __init__(self, version_args, image_version, offline_mode):
+        self.version_args = version_args
+        self.image_version = image_version
+        self.offline_mode = offline_mode
+        self.host = None
+        self.proxy_bind_addr = None
+        self.bundle_http_port = None
+        self.proxy_ports = None
+
+    def configure(self, args, run_result):
```

Comment:
The name `configure` is quite general and does not indicate when it is called. What about `postConductrStart`?